### PR TITLE
Integrate CRAB

### PR DESCRIPTION
**Pull request by nbittich**

### CODE REVIEW

File: app/components/address-register-selector.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    if (addressSuggestion) {
+      const addresses = yield this.addressRegister.findAll(addressSuggestion);
+      if (!this.sourceCrab) {
+        this.sourceCrab = yield this.store.findRecord(
+          'concept',
+          'e59c97a9-4e95-4d65-9696-756de47fbc1f'
+        );
+      }
+      // TODO: this should probably be fixed in the API itself (, if possible)
```
Comment (nbittich) [inline app/components/address-register-selector.js:39]: it's crab itself i think that has duplicates (uri are differents if i remember, but address seems to be exactly the same)

**Review by nbittich — COMMENTED** [review]
lgtm

**Comment by Windvis** [comment]
> it's crab itself i think that has duplicates (uri are differents if i remember, but address seems to be exactly the same)

Yea, but the backend service can do the filtering already, right?

> lgtm
Cool, I'll rebase on master and do some squashing

**Comment by Windvis** [comment]
@nbittich I added the co-authored comment (with `nordine@bittich.be` as email) but for some reason it doesn't detect your Github account (it only shows your name). Do you use a different email address by any chance?

**Comment by nbittich** [comment]
> @nbittich I added the co-authored comment (with `nordine@bittich.be` as email) but for some reason it doesn't detect your Github account (it only shows your name). Do you use a different email address by any chance?

i think my email address on github is contact@bittich.be

**Comment by nbittich** [comment]
> @nbittich ah, I see. you might want to add `nordine@bittich.be` to your list of emails in the settings. It seems your commits are signed with that email address so at the moment none of those are linked to your account 😄.

ah didn't know that, I added it now

**Pull request merged**